extensions/amp-ad-exit/0.1/config.js

```diff
@@ -165,10 +167,10 @@ function assertTarget(name, target, config) {
  * @return {string} The vendor's iframe URL
  */
 export function assertVendor(vendor) {
-  user().assert(ANALYTICS_CONFIG &&
-      ANALYTICS_CONFIG[vendor] &&
-      ANALYTICS_CONFIG[vendor]['transport'] &&
-      ANALYTICS_CONFIG[vendor]['transport']['iframe'],
+  user().assert(ANALYTICS_IFRAME_TRANSPORT_CONFIG &&
```

Comment:
Any reason you need to check that ANALYTICS_IFRAME_TRANSPORT_CONFIG exists?  I don't see when it couldn't...

Reply:
You're right - fixed.